### ISSUE TITLE
Roughly how many pairs are needed??

### ISSUE THREAD

**Issue by robbiebarrat** [keithito/tacotron#82]
Can I get by with only ~30 minutes of audio?

**Comment by keithito** [comment]
30 minutes is probably not enough data. However, if you can find another dataset in the same language, you can probably get good results by training first on the large dataset and then fine-tuning on your own, as was done here:
https://github.com/keithito/tacotron/issues/10#issuecomment-343015698

**Comment by robbiebarrat** [comment]
@keithito awesome - i suspected that this might be the answer. I got really good results doing something similar with GANs when I only had ~50 images in one class and ~14000 in another.

Thanks so much. Do you know of any good datasets spoken by a male in english?

**Comment by keithito** [comment]
Blizzard 2012 is a male reading Mark Twain books.
http://www.cstr.ed.ac.uk/projects/blizzard/data.html

**Comment by keithito** [comment]
Also, the person who did the Obama voice trained first on the [LJ Speech Dataset](https://keithito.com/LJ-Speech-Dataset/), which is a female speaker, so that might be an option too.

**Comment by robbiebarrat** [comment]
@keithito oh cool - do you know how much data you need to convince a pre-trained network to talk differently?? like if i have 15 minutes of super high quality audio / labels - will that cut it? Overall i have about 4 or 5 hours of audio (i'm trying to train the network to talk like kanye west) - but of those 4 or 5 hours, i have scraped around 15 or 20 minutes of kanye freestyling (from old documentaries) - and i'd like to capture his "rapping" voice more as opposed to his talking voice... Any idea if 20 minutes of this is enough for a pre-trained network?

**Comment by keithito** [comment]
It might have trouble due to the difference in cadence between ordinary speech and rapping... but the only way to know for sure is to try :-)

**Comment by keithito** [comment]
Closing out inactive issues. Please feel free to re-open if there's an update.
